src/script/vco-ie/main_warhammer/wh3_main_ogr_goldtooth/missions.lua

```diff
@@ -54,7 +54,7 @@ local missions = {
 				objective
 				{
 					override_text mission_text_text_vco_ogr_gre_2_overtyrants_reign_1;
-					type CONTROL_N_PROVINCES_INCLUDING;
+					type CONTROL_N_PROVINCES_FROM;
```

Comment:
Key doesn't exist. We have to change it to `CONTROL_N_REGIONS_FROM`, inclduing changes to the targets and the docs. Remove it from here and move it to the backlog.

Reply:
Remove it? Do you no longer intend to issue a hotfix for these?

Reply:
I have two hours an a half left before I leave for good. I prefer a hotfix that fixes 2/4 problems than one than arrives late.